.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+174e4e4d86025cfb9590e014438194c362f18bd7
```

Comment:
Should not be committed, right ?

Reply:
This file is meant to be commited :)

It lists all commits we want to ignore when doing `git blame` on the code. The idea is to list commits that are making noise in history, like prettier-specific commits.